### PR TITLE
`browsingContext` events + `browsingContext.navigate` `wait` param.

### PR DESCRIPTION
**Pull request by sadym-chromium**
1. Emit `browsingContext.load` when `Page.lifecycleEvent` fired with `name="load"`.
2. Emit `browsingContext.domContentLoaded` when `Page.lifecycleEvent` fire with `name="DOMContentLoaded"`.
3. Added and implement `interactive` and `complete` params to `browsingContext.navigate` command.

### CODE REVIEW

File: src/bidiMapper/domains/context/context.ts [hunk 1]
```diff
@@ -63,6 +63,22 @@ export class Context {
...
+  }
+
+  private _initializeEventListeners() {
+    this._handleDomContentEvent();
+  }
+
+  private _handleDomContentEvent() {
+    this._cdpClient.Page.on('domContentEventFired', async (params) => {
+      await this._bidiServer.sendMessage({
+        method: 'browsingContext.domContentLoaded',
```
Comment (bwalderman) [inline src/bidiMapper/domains/context/context.ts:76]: CDP `Page.lifecycleEvent` fires both "DOMContentLoaded" and "load" events. We could possibly use the `frameId` and/or `loaderId` properties to implement the BiDi `navigation` info.

Comment (sadym-chromium) [reply src/bidiMapper/domains/context/context.ts]: Good point, thanks!
Fixed and implemented `browsingContext.load` event.

File: src/bidiMapper/domains/context/context.ts [hunk 2]
```diff
@@ -63,6 +63,35 @@ export class Context {
...
+      if (params.name === 'DOMContentLoaded')
+        await this._bidiServer.sendMessage({
+          method: 'browsingContext.domContentLoaded',
+          params: {
+            context: this._contextId,
+            navigation: params.loaderId,
+          },
+        });
+
+      if (params.name === 'load')
```
Comment (bwalderman) [inline src/bidiMapper/domains/context/context.ts:86]: Nit: This can be an `else if`.

File: src/bidiMapper/domains/context/context.ts [hunk 3]
```diff
@@ -121,29 +146,48 @@ export class Context {
...
+        cdpNavigateResult.loaderId!
+      );
+
+      return {
+        navigation: cdpNavigateResult.loaderId,
+        url: url,
+      };
+    }
+
+    if (wait === 'complete') {
```
Comment (bwalderman) [inline src/bidiMapper/domains/context/context.ts:161]: This can also be `else if`.

**Review by bwalderman — APPROVED** [review]
LGTM with minor suggestions.

File: src/bidiMapper/domains/context/context.ts (hunk 3 above)
Comment (sadym-chromium) [reply src/bidiMapper/domains/context/context.ts:161]: replaced with `switch - case`

File: src/bidiMapper/domains/context/context.ts (hunk 2 above)
Comment (sadym-chromium) [reply src/bidiMapper/domains/context/context.ts:86]: replaced with `switch - case`

**Pull request merged**